ietf/settings/base.py

```diff
@@ -175,6 +176,12 @@
 }
 
 
+WAGTAILMARKDOWN = {
+    # We probably want to configure this more, see
+    # https://github.com/torchbox/wagtail-markdown
+    "autodownload_fontawesome": True,
```

Comment:
Default configuration works fine.
I'm happy to leave this as it is and allow people test this on the development server and make any tweaks based on the comments.